src/View/Landing.elm

```diff
@@ -0,0 +1,54 @@
+module View.Landing exposing (view)
+
+import Copy.Keys exposing (Key(..))
+import Copy.Text exposing (t)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Markdown
+import Message exposing (Msg(..))
+import Route exposing (Route(..))
+
+
+view : Html Msg
+view =
+    div [ class "landing container-fluid" ]
+        [ div [ class "container" ]
+            [ div [ class "row my-5" ]
+                [ div [ class "col-12" ]
+                    [ h1 [ class "d-block mx-auto text-center" ]
+                        [ text (t LandingHeader)
+                        ]
+                    ]
+                , Markdown.toHtml [ class "col-12 mt-4" ]
+                    -- join each string with newlines so Markdown.toHtml can insert paragraphs
+                    -- could be joined into one string in Text.elm instead
+                    (t LandingParagraph1
+                        ++ "\n\n"
+                        ++ t LandingParagraph2
+                    )
+                ]
+            , div [ class "row my-4" ]
+                [ iframe
```

Comment:
I made a video component but let's merge like this and then get it combined with mine?